transformStream.js

```diff
@@ -1,5 +1,5 @@
 import {parse} from './tXml.js'
-const through2 = require('through2');
+import through2 from 'through2';
```

Comment:
It's best to avoid mixing import and require statements because it can confuse bundlers. Rollup will convert a top-level import to a require when the target format is cjs, and keep as an import for the esm target.

As a side note, Node's module resolution is allows for imports of commonjs modules, so import through2 from 'through2' works for a Node project with "type": "module" although through2 is commonjs.